exampleSite/content/fragments/_index/list.md

```diff
@@ -0,0 +1,9 @@
++++
+date = "2018-07-06"
+fragment = "list"
```

Comment:
Awesome use of the list fragment here. Still looks a bit bare bones, so we probably should iterate on the design in a follow on PR.